cpp/roaring64map.hh

```diff
@@ -493,6 +493,8 @@ class Roaring64Map {
         return iter->second.contains(lowBytes(x));
     }
 
+    // TODO: implement `containsRange`
```

Comment:
Sure. I would like to implement these along with a `...Closed` version in this PR.

Reply:
My mind changed. I found that it should be handled in another issue. They require new tests. Let's focus on consistency instead of functionalities in this PR.

Reply:
It is probably better to have more PRs. Small PRs are easier to assess.

Reply:
This should be removed then.